taxcalc/tbi/tbi.py

```diff
@@ -177,7 +177,7 @@ def run_nth_year_gdp_elast_model(year_n, start_year,
 
     # calculate gdp_effect
     fyear = check_years_return_first_year(year_n, start_year, use_puf_not_cps)
-    if (start_year + year_n) > fyear:
+    if year_n > 0 and (start_year + year_n) > fyear:
         # create calc1 and calc2 calculated for year_n - 1
```

Comment:
@martinholmer what condition are you checking here `if (start_year + year_n) > fyear:`?

Reply:
year_n can be zero even if (start_year + year_n) is greater than fyear,
which is equal to the value returned by this function:
```
def check_years_return_first_year(year_n, start_year, use_puf_not_cps):
    """
    Ensure year_n and start_year values are valid given input data used.
    Return value of first year, which is maximum of first records data year
    and first policy parameter year.
    """
```

Reply:
OK, I see. Thanks.